studybuddies/src/app/components/login/login.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167323)
```suggestion
import { ComponentFixture, TestBed } from "@angular/core/testing";
```